modules/apps/batch-engine/batch-engine-test/src/testIntegration/java/com/liferay/batch/engine/internal/test/BatchEngineImportTaskExecutorTest.java

```diff
@@ -108,7 +111,11 @@ public void testCreateBlogPostingsFromInvalidCSVFile() {
 
 		String content = sb.toString();
 
-		try {
+		try (CaptureAppender captureAppender =
```

Comment:
@ivicac you need to assert the expected error log after capturing it, otherwise in case the code actually breaks and generates some unexpected error log, this test will still pass. Please apply that and resend.